dss/operations/__init__.py

```diff
@@ -0,0 +1,103 @@
+"""
+This file provides an interface to build the DSS operations CLI, parts of which may be
+executed "in the cloud", via Lambda-SQS integration.
+
+Commands are defined with a target-action model. While command arguments configured at the
+target level are moved into the action arguments.
+
+e.g
+```
+dispatch = DSSOperationsCommandDispatch()
+
+storage = dispatch.target("storage", arguments={"replica": dict(choices=["dev", "integration", "staging", "prod"])})
```

Comment:
`replica` should be `--replica` for the example execution command I think.

Reply:
Correct, fixed!